pinot-core/src/main/java/org/apache/pinot/core/operator/combine/BaseCombineOperator.java

```diff
@@ -55,27 +55,21 @@
   protected final QueryContext _queryContext;
   protected final ExecutorService _executorService;
   protected final long _endTimeMs;
-  protected final int _numThreads;
+  protected final int _numTasks;
```

Comment:
Can you add a comment on what the `task` would mean in this context? (e.g. the unit of parallelism)

Reply:
Added